views/partials/protect_yourself.hbs

```diff
@@ -5,9 +5,10 @@
 {{else}}
   <p class="demi">The first step to keeping your online accounts safe is knowing what you’re up against. Enter your email to find out if your accounts have been compromised.</p>
 {{/if}}
-<form action="/scan" method="post">
+<form action="/scan" class="email-scan" method="post">
     <div class="input-group">
         <input class="input-group-field email-to-hash" type="email" name="email" placeholder="Enter Email Address">
```

Comment:
We should get rid of the name attribute on this one, otherwise its value will be sent when the form is submitted.

Reply:
☝️ great catch!
Also, we'd need to make that change in the other 2 email form templates (home.hbs and partials/what_to_do.hbs).

Reply:
Updated to remove the `name` attribute from the forms.